### ISSUE TITLE
Problem with getting low mAP

### ISSUE THREAD

**Issue by jitesh321** [AlexeyAB/darknet#7581]
I trained my object detection model with both  Yolov4 and Yolov4 tiny but in Yolo v4 I am getting very Low mAP i don't know why but I am using the same dataset in both the models but why it's
![chart_yolov4_tiny](https://user-images.githubusercontent.com/70311549/113666432-e9816c00-96cc-11eb-8b99-ddddb48c9413.png)
[yolov4_tiny.txt](https://github.com/AlexeyAB/darknet/files/6262468/yolov4_tiny.txt)

 not giving good mAP with my dataset.
Even in yolov4-tiny, i am not getting a constant mAP as my loss is decreasing but my mAP is somewhat decreasing or remaining constant. can u tell me if this problem with my training parameter or with my dataset?

**Comment by stephanecharette** [comment]
To me this looks like a normal graph.  I don't know why you would call it "low".

**Comment by jitesh321** [comment]
but the detection is not working that way I don't know why but even after 78% mAP it's giving a False Positive most of the time

**Comment by stephanecharette** [comment]
What do your training images look like, and what does your inference image look like?

**Comment by jitesh321** [comment]
I am a newbie to this computer vision and Deep learning so I think that this might be low that's why detection is not working in the correct way

**Comment by stephanecharette** [comment]
No, that graph looks good.  Your problem is related to something else.  If you are new to this, my guess is your training images and too dissimilar.

**Comment by stephanecharette** [comment]
Join the discord if you want more help:  https://discord.gg/zSq8rtW
